Bio/SeqRecord.py

```diff
@@ -116,6 +115,13 @@ def update(self, new_dict):
             self[key] = value
 
 
+SeqType = Union["Seq", "MutableSeq"]
```

Comment:
Rather than defining this in the public API here, I'd leave the ``Union["Seq", "MutableSeq"]`` in the function signatures. We might later want to define this union in ``Bio/Seq.py`` but that's a discussion for another day...